File: python/Fixed Dynamic.py
# Fixed Dynamic Array
# This code demonstrates how to create a fixed-size dynamic array in Python
# # using a list and how to initialize it with a specific size.
# The array is initialized with zeros, and then each element is set to its index.
# This is a simple example of a fixed dynamic array in Python.
n = int(input("Enter size: "))
arr = [0] * n # Create a fixed-size dynamic array
for i in range(n):
    arr[i] = i
print(arr)

File: c++/Fixed Dynamic.cpp
// Fixed Dynamic Array
// This code demonstrates the use of a fixed-size array in C++.
// It uses a variable-length array (VLA) to allocate memory on the stack.


#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter size: ";
    cin >> n;
    int arr[n]; // variable-length array
    for (int i = 0; i < n; i++) arr[i] = i;
    for (int i = 0; i < n; i++) cout << arr[i] << " ";
}
